DQM/Physics/src/TopSingleLeptonDQM_miniAOD.cc

```diff
@@ -373,11 +393,24 @@ void MonitorEnsemble::fill(const edm::Event& event,
   // fill monitoring plots for primary verices
   edm::Handle<edm::View<reco::Vertex> > pvs;
   if (!event.getByToken(pvs_, pvs)) return;
-  unsigned int pvMult = 0;
-  for (edm::View<reco::Vertex>::const_iterator pv = pvs->begin();
+  const reco::Vertex& pver= pvs->front();
+
+  unsigned int pvMult = 0; 
+  if(pvs.isValid()){
+    for (edm::View<reco::Vertex>::const_iterator pv = pvs->begin();
        pv != pvs->end(); ++pv) {
-    if (!pvSelect_ || (*pvSelect_)(*pv)) pvMult++;
```

Comment:
it looks like pvSelect is no longer used. Please remove it

Reply:
L379 including pvSelect_ was removed. New PR was sent.